src/loaders/resourceLoader.test.ts

```diff
@@ -457,8 +457,6 @@ describe("ResourceLoader::deleteSchemaVersion()", () => {
     const logErrorArgs = logErrorStub.getCall(0).args;
     assert.strictEqual(logErrorArgs[0], thrownError);
     assert.strictEqual(logErrorArgs[1], "Error deleting schema version");
```

Comment:
Probably don't need to match the exact sentry context, but making sure it's not nullish would still be good to keep around
```suggestion
    assert.strictEqual(logErrorArgs[1], "Error deleting schema version");
    // would send to Sentry.
    assert.ok(logErrorArgs[3]);
```